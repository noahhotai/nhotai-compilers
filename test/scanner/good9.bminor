fsdf;
sdfasdfdsaf;
sdfasdfsd; p =2 

char;
>
<
== 
-
+
++
--