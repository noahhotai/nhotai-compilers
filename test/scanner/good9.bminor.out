IDENTIFIER     fsdf
SEMICOLON      ;
IDENTIFIER     sdfasdfdsaf
SEMICOLON      ;
IDENTIFIER     sdfasdfsd
SEMICOLON      ;
IDENTIFIER     p
ASSIGN         =
INT LITERAL    0
CHAR           char
SEMICOLON      ;
GREATER THAN   >
LESS THAN      <
EQUALITY       ==
SUBTRACT           -
ADD            +
INCREMENT      ++
DECREMENT      --
